torch/csrc/jit/codegen/fuser/cpu/fused_kernel.cpp

```diff
@@ -45,60 +45,66 @@ constexpr int so_suffix_len = 3;
 constexpr int cpp_suffix_len = 4;
 #endif
 
+intptr_t run(const std::string& cmd);
+
 static bool programExists(const std::string& program) {
   TemplateEnv env;
   env.s("program", program);
   std::string cmd = format(check_exists_string, env);
+#ifdef _MSC_VER
+  return (run(cmd.c_str()) == 0);
```

Comment:
@peterjc123 This one is kinda a bugfix/improvement. On Windows using `system` will not find `cl` correctly. Do you feel comfortable I add this in the PR?

Reply:
Yes, but why?

Reply:
This is addressed in #50085.